src/terrain/routes/schemas/filesystem/tickets.clj

```diff
@@ -0,0 +1,17 @@
+(ns terrain.routes.schemas.filesystem.tickets
+  (:use [common-swagger-api.schema :only [describe]])
+  (:require [common-swagger-api.schema.data.tickets :as schema]
+            [schema.core :as s]
+            [schema-tools.core :as st]))
+
+;; Convert the keywords in the `mode` param to strings,
+;; so that the correct param format is passed through to the apps service.
+;; Remove `for-job` param.
+;; FIXME: Make `public` param optional (for backwards compatibility).
+(s/defschema AddTicketQueryParams
+  (-> schema/AddTicketQueryParams
+      (st/dissoc :for-job)
```

Comment:
I thought the `data-info` query param `for-job` was for internal use (by the `apps` service) only, so I'm not exposing it in the add or delete ticket endpoints in `terrain`.

Reply:
I _think_ that's accurate. We can always update it later if we need to.